contracts/ethereum/contracts/Token.sol

```diff
@@ -169,20 +155,19 @@ contract Token is IERC20, ITransferWithBeneficiary {
         string memory _symbol,
         address _emergencyCouncil,
         address _operator,
-        address _workerConfig
+        address _registry
```

Comment:
We need to have the registry set up properly that we can call already keyed into everything before we do this deploy

Reply:
Should be renamed to _oracle again

Reply:
We need to kick off Factory for the new deployments/upgrades